support/rally.js

```diff
@@ -27,8 +32,11 @@ export default class Rally {
    *        Whether or not to initialize Rally.js in developer mode.
    *        In this mode we ignore problems when communicating with
    *        core add-on.
+   *
+   * @param {Function} stateChangeCallback
+   *        A function to call when the study is paused or running.
```

Comment:
I meant to bring this up, thanks... since we're using positional arguments (as opposed to an options object or parameter desctructuring etc) then it seems like we'd need to enforce all of the options (throw if any argument is not provided or an incorrect type)

Reply:
Well, my previous comment assumes we want to insist that these are required and not optional arguments. If they are optional, then making sure they are the expected type and such is fine and I'll just do that for now.

Reply:
Oh, it turns out that jsdoc has support for callback functions: https://jsdoc.app/tags-callback.html

Reply:
Ugh, it turns out that the jsdoc `@callback` needs to be registered elsewhere in the file. It's probably just fine for auto-generated docs, but since it won't be above the initialize function here I don't think it's useful. I went ahead and documented it in the jsdoc header for initialize, instead.